ouroboros-consensus-diffusion/changelog.d/20250130_100651_tdammers_mlocked_kes_rebase.md

```diff
@@ -0,0 +1,3 @@
+### Non-Breaking
+
+- Use registry to ensure block forging threads are finalized in node kernel.
```

Comment:
"Use registry" is a bit ambiguous. I would instead say
```suggestion
- Ensure that block forging threads erase their keys on shutdown in node kernel.
```

Reply:
It's not necessarily just erasing; finalizing may also involve shutting down a KES agent connection in the future.